src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.hpp

```diff
@@ -171,8 +171,11 @@ class TDatabaseFeedManager final : public Observer<nlohmann::json&>
 
                 auto orchestrationLambda = [&](const nlohmann::json& resource)
                 {
-                    auto eventContext = std::make_shared<EventContext>(
-                        EventContext {.message = message, .resource = resource, .cvesDatabase = m_cvesDatabase});
+                    auto eventContext =
+                        std::make_shared<EventContext>(EventContext {.message = message,
+                                                                     .resource = resource,
+                                                                     .cvesDatabase = m_cvesDatabase,
+                                                                     .translationsDatabase = m_translationsDatabase});
```

Comment:
Coding style: Only `eventDecoder` requires access to `m_translationsDatabase`, could we put in that constructor instead of the context? The `storeModel` class does a similar thing.